examples/apps/collaborative-textarea/package.json

```diff
@@ -38,16 +38,16 @@
 		"webpack:dev": "webpack --env development"
 	},
 	"dependencies": {
-		"@fluid-example/example-utils": ">=2.0.0-internal.3.4.0 <2.0.0-internal.4.0.0",
-		"@fluid-experimental/react-inputs": ">=2.0.0-internal.3.4.0 <2.0.0-internal.4.0.0",
-		"@fluidframework/aqueduct": ">=2.0.0-internal.3.4.0 <2.0.0-internal.4.0.0",
+		"@fluid-example/example-utils": "^2.0.0-internal.3.4.1",
```

Comment:
what happened to the dependency range? Doesn't "^" not work with our versioning?